client/pages/Home/Components/BookDetails.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import notebook from '@images/notebook.png';
+
+const BookDetails = () => {
```

Comment:
Rename this file to `Book`, move it to `./client/components/Book` since it will be used in many more pages than the home.